dashboard/db/migrate/20220316125751_applied_at_column.rb

```diff
@@ -0,0 +1,15 @@
+class AppliedAtColumn < ActiveRecord::Migration[5.2]
+  def change
+    reversible do |dir|
+      dir.up do
+        add_column :pd_applications, :applied_at, :datetime
+        Pd::Application::ApplicationBase.all.each do |application|
+          application.update_column(:applied_at, application.created_at)
+        end
+      end
+      dir.down do
+        remove_column :pd_applications, :applied_at, :datetime
```

Comment:
this is totally fine how you have it, but it's a little more complicated than it needs to be. for next time you can try something like:
```suggestion
    add_column :pd_applications, :applied_at, :datetime
    reversible do |dir|
      dir.up do
        Pd::Application::ApplicationBase.all.each do |application|
          application.update_column(:applied_at, application.created_at)
        end
```
when you do add_column (and many other railsy operations) inside the `change` method, then it takes care of undoing them for you... so you usually only need the `dir.up` block for things that are not reversible rails commands. but the way you have it is still fine since it is still clear what you're doing.

Reply:
Oo yes, we talked about that. Makes sense––thanks!